.github/workflows/security.yml

```
@@ -0,0 +1,171 @@
+
+  dependency-review:
+    name: Dependency Security Review
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Dependency Review
+        uses: actions/dependency-review-action@v3
```

actionlint (1.7.7)

the runner of "actions/dependency-review-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)